hack/jenkins/common.sh

```diff
@@ -289,6 +301,35 @@ elapsed=$min.$sec
 description="completed with ${status} in ${elapsed} minute(s)."
 echo $description
 
+echo ">> Copying ${TEST_OUT} to gs://minikube-builds/logs/${MINIKUBE_LOCATION}/${JOB_NAME}out.txt"
+gsutil -qm cp "${TEST_OUT}" "gs://minikube-builds/logs/${MINIKUBE_LOCATION}/${JOB_NAME}out.txt"
+
+
+echo ">> Attmpting to convert test logs to json"
+rm ${JSON_OUT} || true # clean up build-reruns
```

Comment:
Enclose all variables in double quotes to prevent unintentional expansion issues: `rm "${JSON_OUT}"`

https://google.github.io/styleguide/shell.xml?showone=Variable_expansion#Variable_expansion